### PR TITLE
Similar listings

### PR DESCRIPTION
**Pull request by mleeBoeing**
Display Similar Listing on Listing page

 

Acceptance Criteria:
Similar Listings are displayed on the listing page per the mock ups.

### CODE REVIEW

**Comment by blynch11p** [comment]
I see the same listing in Similar that I'm currently viewing.  I think this was handling in 2.0.  I guess not by the backend.  Can you look at removing it through the Frontend?

I can clean up the rest of the styles later

**Comment by mleeBoeing** [comment]
Added the filter

**Comment by blynch11p** [comment]
Thanks for fixing the duplicate.  
Last comment, for now, is a SimilarListingTile at all different from a ListingTile?  Can you just reuse the ListingTile?

**Comment by mleeBoeing** [comment]
Removed SimilarListingTile

**Comment by tdressel** [comment]
Displays vertical on the right hand side. :+1:

File: src/actions/listing.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import * as ProfileActions from 'actions/creators/profile';
 import * as actions from 'actions/creators/listing';
 import * as listingService from 'services/listing';
 import * as imageService from 'services/image';
-import { Listing, Review, Activity } from 'services/serializers/listing';
+import { Listing, Review, Activity, SimilarListings } from 'services/serializers/listing';
```
Comment (irtiza-rizvi) [inline src/actions/listing.js]: import is not necessary, otherwise looks good :+1:

**Comment by blynch11p** [comment]
:+1:

**Pull request merged**